internal/config/context.go

```diff
@@ -94,6 +96,23 @@ func CurrentContext() (endpoint Endpoint, user *User) {
 	return
 }
 
+// GetCurrentContextInfo returns endpoint and basic auth info
+// associated with current context
+func GetCurrentContextInfo() (server string, username string, password string) {
+	endpoint, user := CurrentContext()
+	server = fmt.Sprintf("%s,%d", endpoint.Address, endpoint.Port)
```

Comment:
>,%d [](http://example.com/codeflow?start=25&length=3)

do endpoints always have a port? What about people using non-tcp connections?

Reply:
how about people using named instances?

Reply:
As per the help summary for add-endpoint, it needs --address and --port defaults to 1433 if, not specified

Reply:
pls open an issue to track support for non-tcp connections, as they haven't gone away from real enterprise deployments yet